spanner/cloud-client/src/main/java/com/example/spanner/SpannerSample.java

```diff
@@ -1155,6 +1210,256 @@ public Void run(TransactionContext transaction) throws Exception {
+            null);
+    try {
+      // Initiate the request which returns an OperationFuture.
+      op.get();
+      System.out.println("Created Venues table in database: [" + id + "]");
+    } catch (ExecutionException e) {
+      // If the operation failed during execution, expose the cause.
+      throw (SpannerException) e.getCause();
+    } catch (InterruptedException e) {
+      // Throw when a thread is waiting, sleeping, or otherwise occupied,
+      // and the thread is interrupted, either before or during the activity.
+      throw SpannerExceptionFactory.propagateInterrupt(e);
+    }
+  }
+  // [END spanner_create_table_with_datatypes]
+
+  // [START spanner_insert_datatypes_data]
+  static void writeDatatypesData(DatabaseClient dbClient) {
+    List<Mutation> mutations = new ArrayList<>();
+    for (Venue venue : VENUES) {
+      mutations.add(
+          Mutation.newInsertBuilder("Venues")
+              .set("VenueId")
+              .to(venue.venueId)
+              .set("VenueName")
+              .to(venue.venueName)
+              .set("VenueInfo")
+              .to(venue.venueInfo)
+              .set("Capacity")
+              .to(venue.capacity)
+              .set("AvailableDates")
+              .to(venue.availableDates)
+              .set("LastContactDate")
+              .to(venue.lastContactDate)
+              .set("OutdoorVenue")
+              .to(venue.outdoorVenue)
+              .set("PopularityScore")
+              .to(venue.popularityScore)
+              .set("LastUpdateTime")
+              .to(Value.COMMIT_TIMESTAMP)
+              .build());
+    }
+    dbClient.write(mutations);
+  }
+  // [END spanner_insert_datatypes_data]
+
+  // [START spanner_query_with_array_parameter]
+  static void queryWithArray(DatabaseClient dbClient) {
+    Value exampleArray = Value.dateArray(Arrays.asList(
+        Date.parseDate("2020-10-01"), 
+        Date.parseDate("2020-11-01")));
+
+    Statement statement =
+        Statement.newBuilder(
+                "SELECT VenueId, VenueName, AvailableDate FROM Venues v, \n"
+                    + "UNNEST(v.AvailableDates) as AvailableDate \n"
+                    + "WHERE AvailableDate in UNNEST(@availableDates)")
+            .bind("availableDates")
+            .to(exampleArray)
+            .build();
+    try (ResultSet resultSet = dbClient.singleUse().executeQuery(statement)) {
+      while (resultSet.next()) {
+        System.out.printf(
+            "%d %s %s\n",
+            resultSet.getLong("VenueId"),
+            resultSet.getString("VenueName"),
+            resultSet.getDate("AvailableDate"));
+      }
+    }
+  }
+  // [END spanner_query_with_array_parameter]
+
+  // [START spanner_query_with_bool_parameter]
+  static void queryWithBool(DatabaseClient dbClient) {
+    boolean exampleBool = true;
+    Statement statement =
+        Statement.newBuilder(
+                "SELECT VenueId, VenueName, OutdoorVenue FROM Venues \n"
```

Comment:
Does spanner require the \n ?  if not, why is it here?

Reply:
Removed.